doc/release-30635.md

```diff
@@ -1,4 +1,5 @@
 Updated RPCs
 ------------
 
-- The waitfornewblock RPC takes an optional `current_tip` argument. (#30635)
+- The waitfornewblock takes an optional `current_tip` argument. It is also no longer hidden. (#30635)
```

Comment:
In commit "rpc: unhide waitfor{block,newblock,blockheight}" (067acb5907b5b135cff702da13a1f64a02d64110)

Maybe s/takes/now takes/ to be clear this is a change